src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
-export interface Figure {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
 }
 
-export class Triangle {
+function area(square: number): number {
+  return Math.floor(square * 100) / 100;
+}
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides: number[] = [a, b, c].sort((sideA, sideB) => sideB - sideA);
+
+    if (a < 0 || b < 0 || c < 0) {
```

Comment:
you use check for valid arguments three times in your code, create separate function